label_studio/io_storages/base_models.py

```diff
@@ -880,8 +897,8 @@ class ImportStorageLink(models.Model):
     row_index = models.IntegerField(null=True, blank=True, help_text='Parquet row index, or JSON[L] object index')
 
     @classmethod
-    def n_tasks_linked(cls, key, storage):
-        return cls.objects.filter(key=key, storage=storage.id).count()
```

Comment:
why not keep around this class method as well, but change the signature to `cls, key_set, storage`? This could be used to make the new code a bit DRYer

Reply:
Since it's used in one place with `key=`, and another place with `keys__in=`, I didn't want to combine those, it seemed premature (even though `keys__in=[key]` is a valid use I felt it would increase rather than reduce confusion) - especially since we're going to be getting rid of one.